dusty/commands/run.py

```diff
@@ -93,10 +93,10 @@ def restart_apps_or_services(app_or_service_names=None, sync=True):
         if app_or_service_names:
             specs = spec_assembler.get_assembled_specs()
             specs_list = [specs['apps'][app_name] for app_name in app_or_service_names if app_name in specs['apps']]
-            rsync.sync_repos_by_specs(specs_list)
+            repos_list = [spec['repo'] for spec in specs_list if spec['repo']]
+            nfs.update_nfs_with_repos(repos_list)
         else:
-            rsync.sync_repos(spec_assembler.get_all_repos(active_only=True, include_specs_repo=False))
-
+            nfs.update_nfs_with_repos(spec_assembler.get_all_repos(active_only=True, include_specs_repo=False))
```

Comment:
I think it might make sense to move all of the NFS handling shit to `up` and `test`. We can talk about it. Basically, if you do anything during restart, you're going to yank NFS mounts away from running processes, which seems like a recipe for weirdness.